site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "High Availability Agones"
+date: 2023-02-10
+weight: 20
+description: >
+  Learn how to configure your Agones services for high availability.
+publishDate: 2023-02-28
+---
+
+## High Availability for Agones Controller
+
+When `SplitControllerAndExtensions` is enabled, the `agones-controller` responsibility is split up into `agones-controller`, which enacts the Agones control loop, and `agones-extensions`, which acts as a service endpoint for webhooks and the allocation extension API. Splitting these responsibilities allows the `agones-extensions` pod to be horizontally scaled, making the Agones control plane highly available.
```

Comment:
🤔 is it just that it's horizontally scalable, or it it more that it's more resilient to disruption, and therefore can handle node upgrades etc with minimal disruption? (or eventually anyway. Do we have a pod disruption budget?)

Reply:
It should be both horizontally scalable and also more resilient to disruption. And yup, we do have a PDB for extensions currently with `minAvailable` set to 1.

Reply:
Should we add that there is a PDB for `agones-extensions` to the doc?

Reply:
Nice! Then I think we should highlight that - especially since (I assume) the main focus on this work is actually reliability and fault tolerance, not really horizontal scalability - no?

Reply:
> Should we add that there is a PDB for `agones-extensions` to the doc?

I think we should - it goes to the _why_ this exists, not just the _how_.

Reply:
Added resiliency to the docs and also did a *almost* literal highlight. 😅

Reply:
PDB is not part of the `Extensions Pod Configuration` section.

Reply:
I had asked @chiayi to take it out but mostly because I thought it was more detail than necessary. :shrug:

Reply:
Hmm. I especially don't think the PDB belongs in this intro section? But maybe? I'm happy if it ends up here but it seems more like an implementation detail that doesn't need to be laid out here.